filesystem.go

```diff
@@ -152,11 +152,6 @@ func (fs *filesystem) createAllResources(pod Pod) error {
 }
 
 func (fs *filesystem) storeFile(path string, data interface{}) error {
-	_, err := os.Stat(path)
```

Comment:
Hi @sboeuf - I've looked at this again and I don't think the `podlockFile` `Stat()` call can be removed. Also, since it's an empty file it's going to be created atomically anyhow.

Summary: I don't think there are any further changes required for this PR :-)